### PR TITLE
Completed ticket 6

### PR DESCRIPTION
**Pull request by Storme419**

### CODE REVIEW

**Review by HeatherEver — COMMENTED** [review]
Really nice work Storme - I would strongly consider moving all the comments logic into a separate component that ArticlePage so it is cleaner and more readable 
and this is just a suggestion but when an article has no comments it could be nice to display a message telling the user that :)

**Review by Beth-McCormack — CHANGES_REQUESTED** [review]
Nice work so far! 
Currently when there is an article with no comments (id 37) the comment section will continue to say 'loading comments'. Please add some feedback for the user for when theres no comments and resubmit when this is done

File: src/Components/CommentSection.jsx [hunk 1]
```diff
@@ -12,14 +12,18 @@ const CommentSection = ({id}) => {
         .then((res) => {
             setArticleComments(res)
             setLoading(false)
+        }).catch((er) => {
+            console.log(er)
+            setArticleComments([])
```
Comment (mezzDavies) [inline src/Components/CommentSection.jsx:17]: This setting the comments to an empty array shouldn't be necessary. Your API should return an empty array if there are no comments for a particular article. This is a work around. 😄  Don't worry about fixing this on your backend project right now though but just understand it is not quite right. 👍

**Review by mezzDavies — APPROVED** [review]
This is approved but your implementation is actually a work-around because your API is not quite functioning as it should. See my other comment. All good just something you might need to fix.

**Pull request merged**